libraries/AP_DDS/README.md

```diff
@@ -103,6 +103,11 @@ Follow the steps to use the microROS Agent
 
   - https://docs.ros.org/en/humble/Installation/Ubuntu-Install-Debians.html
 
+- Install geographic_msgs
+  ```bash
+  sudo apt install ros-humble-geographic-msgs
```

Comment:
Later, once the testing PR that adds colcon support to this repo is added, dependencies can be added there to the package.xml. For now, this is fine.

Reply:
Agreed, we can just rely on rosdep when package.xml is ready